patches/tModLoader/Terraria/ID/AchievementID.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace Terraria.ID
+{
+	/// <summary>
+	///		Constant IDs for vanilla achievements, provided by tModLoader.
+	/// </summary>
+	public static class AchievementID
```

Comment:
maybe do this as an enum? Then you can pick the Achievement ID with AchievementID.Timber for example

Reply:
No.